pkg/controllers/dnspolicy/dnspolicy_controller.go

```diff
@@ -69,6 +69,9 @@ type DNSPolicyReconciler struct {
 //+kubebuilder:rbac:groups=kuadrant.io,resources=dnspolicies/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=kuadrant.io,resources=dnspolicies/finalizers,verbs=update
 //+kubebuilder:rbac:groups=cluster.open-cluster-management.io,resources=managedclusters,verbs=get;list;watch
+// +kubebuilder:rbac:groups=gateway.networking.k8s.io,resources=gateways,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Probably doesn't need some of these permissions, can't think it will ever be using create and delete anyway. I guess it might update status.

Reply:
updated